sample/derivative.cc

```diff
@@ -284,7 +284,7 @@ int main(int argc, char** argv) {
   // print help if not enough arguments
   if (argc<2) {
     std::printf("\n Arguments: modelfile [nthread niter nwarmup nepoch nstep eps]\n\n");
-    return 1;
+    return EXIT_FAILURE;
```

Comment:
You chose to return `EXIT_SUCCESS` for the equivalent branch ("print help") in the other programs, so let's do that here too.